README.md

```diff
@@ -1,25 +1,31 @@
 
-# EC-action-verifier
+# Enterprise Contract GitHub Actions
 
-EC to be a general purpose tool for people who want to build their software securely. It Validate conformance of container images with the Enterprise Contract.
-## Documentation
+## verifier
+"Verify" is a custom GitHub Action made by Enterprise Contract that validates container images for security and compliance. It performs a three-stage validation process:
 
-[EC Documentation](https://enterprisecontract.dev/)
+1. **Signature Verification:** Checks if the image is signed and verifies the signature with a provided public key.
 
+2. **Attestation Verification:** Ensures the image has signed and valid attestations, providing additional information about its source.
 
-## Environment Variables
+3. **Policy Conformance Verification:** Verifies that the image's attestations comply with defined policies, ensuring adherence to organizational standards.
+
+[Read more](https://enterprisecontract.dev/docs/ec-cli/main/ec_validate_image.html#_synopsis) [Read more](https://redhat-appstudio.github.io/book/book/enterprise-contract.html#:~:text=EC%20CLI,or%20violations%20produced)
```

Comment:
This looks a little strange at https://github.com/seanconroy2021/actions/tree/main#readme . Maybe this can become a bullet list with more descriptive link titles, e.g.:

## Read more

* [..](..)
* [..](..)